src/pool/srv_layout.h

```diff
@@ -73,6 +73,9 @@ extern d_iov_t ds_pool_prop_scrub_freq;		/* uint64_t */
 extern d_iov_t ds_pool_prop_scrub_thresh;	/* uint64_t */
 extern d_iov_t ds_pool_prop_svc_redun_fac;	/* uint64_t */
 extern d_iov_t ds_pool_prop_obj_version;	/* uint32_t */
+extern d_iov_t ds_pool_prop_checkpoint_mode;    /* uint64_t */
```

Comment:
[minor] do we really need it uint64? we might have only few modes, maybe uint8 is fine?

Reply:
Yeah, you are right.  Most of this is just copy paste from other properties